### PR TITLE
5 add docker compose file for production

### LINKED ISSUE #5: add docker-compose file for production

### PR DESCRIPTION
**Pull request by Walkmana-25**

### CODE REVIEW

File: Docker/app/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+WORKDIR /go/src/app
+COPY . /go/src/app
+
+RUN CGO_ENABLED=0 go build -o api main.go
+RUN chmod +x api
+
+FROM gcr.io/distroless/static-debian11
+WORKDIR /app
+COPY --from=base /go/src/app/api /app/api
+ENTRYPOINT [ "/app/api" ]
```
Comment (sasanqua-dev) [inline Docker/app/Dockerfile]: please add new line

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      context: .
+      dockerfile: Docker/db_migration/Dockerfile
+    depends_on:
+      db:
+        condition: service_healthy
+    environment:
+      DATABASE_URL: mysql://root:root@db:3306/database
+
+volumes:
+  db-volume:
```
Comment (sasanqua-dev) [inline docker-compose.yml]: please add new line

File: Docker/app/Dockerfile (hunk 1 above)
Comment (Walkmana-25) [reply Docker/app/Dockerfile]: Where should i add?

File: docker-compose.yml (hunk 2 above)
Comment (Walkmana-25) [reply docker-compose.yml]: Where should i add?

**Review by sasanqua-dev — APPROVED** [review]

**Pull request merged**